### PR TITLE
rsweb-8689-app-style-icons

### PR DESCRIPTION
**Pull request by amberRrucker**
RSWEB-8689

Reproducing the "app style" icons on mobile in the ceiling.
Removing the text, increasing the size of the icons.

### CODE REVIEW

File: styleguide/_themes/derek/scss/globalElements/mainnav.scss [hunk 1]
```diff
@@ -344,20 +344,19 @@
...
 
-  .main-nav {
-
-    .navbar-phoneicon,
-    .navbar-chaticon,
-    .navbar-emailicon,
-    .navbar-searchicon {
-      @include make-xs-column(2);
-      border-left: solid 1px $gray-darker;
-      height: 63px;
```
Comment (amberRrucker) [inline styleguide/_themes/derek/scss/globalElements/mainnav.scss]: this was unnecessary code that I didn't clean out when sexy nav was being developed.

File: styleguide/derek/_partials/_ceiling.ejs [hunk 2]
```diff
@@ -5,13 +5,13 @@
           <nav class="ceiling-menuNav">
               <ul class="ceiling-menuInline" role="menu">
                 <li class="ceiling-menuInline-item">
-                  <a class="navbar-phoneicon" title="Sales/Support Call"  data-toggle="modal" data-target="#myModal">Call Us</a>
+                  <a class="navbar-phoneicon" title="Sales/Support Call"  data-toggle="modal" data-target="#myModal"><span class="hidden-xs">Call Us</span></a>
```
Comment (amberRrucker) [inline styleguide/derek/_partials/_ceiling.ejs]: Thinking @WidgetsBurritos is probably going to want this with a more maintainable class name huh?

Comment (WidgetsBurritos) [inline styleguide/derek/_partials/_ceiling.ejs]: We're already using `hidden-xs` further down the file, so I'm not too worried about it here. If we ever refactor this in the future, then maybe we can deal with that then for now, I'm okay with this.

**Comment by runrog** [comment]
@amberRrucker The only weird thing here is the email envelope. It's technically centered in it's div but that extra space is making it look off.. I added a border on the right? maybe bring in some of that right padding? 
screen shot 2017-03-16 at 10 12 29 am

**Comment by amberRrucker** [comment]
@runrog good point. I'll do that real quick like.

**Comment by amberRrucker** [comment]
@runrog fixed the alignment issue.

**Pull request merged**